Gemfile.lock

```diff
@@ -149,5 +149,8 @@ DEPENDENCIES
   rspec
   sinatra (~> 2.1.0)
 
+RUBY VERSION
+   ruby 2.7.0p0
```

Comment:
isn't `ruby-2.7.2` current?

Reply:
It's what I have on my machine locally, I think it's fine for now